### ISSUE TITLE
Issues with PDF in V12

### ISSUE THREAD

**Issue by Olawale1** [frappe/erpnext#19538]
Hi ,

We are encountering issues with PDF in V12 and it seems it's because of the new scrolling feature in print formats. In previous versions, the PDF Print for wider tables would shrink the content to fit in the page. Now, the content gets cut-off! Please see below:

![Scrolling Issue 1](https://user-images.githubusercontent.com/9269337/68525749-31804180-02d5-11ea-9ee7-50925a0a1ecd.png)
![Scrolling Issue 2](https://user-images.githubusercontent.com/9269337/68525750-31804180-02d5-11ea-8023-1eed930d4162.png)

https://discuss.erpnext.com/t/issues-with-pdf-in-v12/54786

Please help fix

Kind regards,

**Comment by ophl55** [comment]
Can you pleased check your wkhtmltopdf version?

`$ wkhtmltopdf --version`

**Comment by Olawale1** [comment]
0.12.5 (with patched qt)

**Comment by ophl55** [comment]
Okay, the version ist correct.

**Comment by Olawale1** [comment]
Yes. I suspect the issue is from the styling for the scrolling feature. I got around it by directly setting the zoom in the print format. Ideally, it should resize automatically
